app/controllers/help_controller.rb

```diff
@@ -1,4 +1,8 @@
 class HelpController < ApplicationController
-	before_filter -> { @css_framework = :bootstrap }
-	layout ->(c) { request.format == :mobile ? "application" : "with_header_with_footer" }
-end
+  before_filter -> { @css_framework = :bootstrap }
+  layout -> (c) { request.format == :mobile ? "application" : "with_header_with_footer" }
+
+  def faq
+    gon.chat_enabled = AppConfig.chat.enabled?
```

Comment:
Looks like gon converts that to headlessCamlCase.

![](http://cloud.aeshna.de/u/mrzyx/screenshots/screenshot_20150222_000933.png)